src/SetCriteria.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Button, Table } from "semantic-ui-react";
+import CriterionEditRow from "./CriterionEditRow";
+import SetCriteriaHeader from "./SetCriteriaHeader";
+import EditCriteriaForm from "./EditCriteriaForm";
+import JudgingRoundSelector from "./judging_round_selector";
+import { cloneCriteriaURL } from "./utils";
+
+class SetCriteria extends React.Component {
+  constructor(props) {
+    super(props);
+    this.judging_round = 56; // change this once app-allocator-setup is calling this page
+    this.cloneCriteria = this.cloneCriteria.bind(this);
+  }
+  state = {
+    judging_round: null
+  };
```

Comment:
remove semi-colons

Reply:
Semicolons are required in our standards. 
https://github.com/masschallenge/standards/blob/master/js_standards.md#432-semicolons-are-required